### PR TITLE
Amy/authflow

### PR DESCRIPTION
**Pull request by amywangg**
update routes for retrieving customer and adding new customer

### CODE REVIEW

File: customer.py [hunk 1]
```diff
@@ -1,35 +1,40 @@
...
-    client_id=request.args.get('client_id')
+def createUser():
+    if request.method == 'POST':
+        params = request.json
+        first_name = params['first_name']
+        last_name = params['last_name']
+        email = params['email']
+        client_id = params['id']
 
     try:
```
Comment (vivvvli) [inline customer.py:18]: good idea implementing try/except logic here!

File: models.py [hunk 2]
```diff
@@ -1,38 +1,43 @@
...
 
 class Budget(db.Model):
     __tablename__ = 'budget'
 
     id = db.Column(db.Integer, primary_key=True)
-    customer_id = db.Column(db.String(100),db.ForeignKey('customer.id'),nullable=False)
-    budget = db.Column(db.Float(),nullable=False)
+    customer_id = db.Column(db.String(100), db.ForeignKey(
+        'customer.id'), nullable=False)
+    budget = db.Column(db.Float(), nullable=False)
```
Comment (vivvvli) [inline models.py:40]: I've noticed a lot of the changes in this commit were formatting and making the code more readable; good work!

**Review by vivvvli — APPROVED** [review]
Great work overall! The code was really easy to understand, I love how you went through and reformatted everything/ spaced it out. Keep it up! 👍

File: models.py (hunk 2 above)
Comment (amywangg) [reply models.py:40]: thanks vivian!

**Pull request merged**